include/wx/object.h

```diff
@@ -372,6 +372,10 @@ class wxObjectDataPtr
 
 class WXDLLIMPEXP_BASE wxObject
 {
+#if wxUSE_EXTENDED_RTTI
+    wxDECLARE_DYNAMIC_CLASS(wxObject);
+#endif
```

Comment:
Sorry but why do we need it here now?

Reply:
You removed it in 95c98a0b5ff71caca6654327bf341719c6587766 0585c96a3bceae440cfe455687e62def82ed8d75 and manually defined `ms_classInfo`.

While RTTI only has `ms_classInfo`, XTI has a lot more static members that now all need to be defined in `object.h`. The simpler solution is to keep using `wxDECLARE_DYNAMIC_CLASS`.

https://github.com/wxWidgets/wxWidgets/blob/ce46d773609154f9e6c663e08aca96e2314db23d/include/wx/xti.h#L437-L452

Reply:
I see, thanks, I should have guessed I must have been broken it myself...